public/index.html

```diff
@@ -41,4 +41,5 @@
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
   </body>
+  <script src="https://kit.fontawesome.com/7777890cf7.js" crossorigin="anonymous"></script> 
```

Comment:
Not a necessity but there is an npm package you can use for font awesome as well.